airflow/providers/atlassian/jira/provider.yaml

```diff
@@ -22,11 +22,12 @@ description: |
     `Atlassian Jira <https://www.atlassian.com/>`__
 
 versions:
+  - 2.0.0
```

Comment:
no need for this here. the release manager decide the version upon release

Reply:
Got it. Removed this change.